### ISSUE TITLE
kleinere Raw-Dateien schreiben

### ISSUE THREAD

**Issue by nFrechen** [fablabcb/CitRad-SensorUnit#7]
z.B. Stündlich neue Dateien anlegen damit man keine 4GB Dateien einlesen muss

**Comment by thorondev** [comment]
Nach je einer Stunde wird nun eine neue Datei geöffnet.

**Comment by nFrechen** [comment]
Die start_time wird in die Datei geschreiben aber die millis werden nicht zurück gesetzt beim Erstellen einer neuen Datei. Wenn man nun versucht die Zeit zu rekonstruieren als start_time + millis hat man einen Zeitversatz.

**Comment by nFrechen** [comment]
Ok, kann man beheben indem man den ersten gespeicherten millisecond Wert von allen Zeitstempeln abzieht.

**Comment by thorondev** [comment]
Ja, das ist immer die Zeit, seit der der Sensor läuft und damit muss man das sowieso immer machen, um auf die korrekte Zeit zu kommen. Da die Datei erst angelegt wird, wenn der erste Datensatz geschrieben wird, entspricht der erste Zeitstempel der Erstellzeit der Datei.
